.github/workflows/screenshots.yml

```diff
@@ -0,0 +1,79 @@
+name: Screenshots
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main    
+
+jobs:
+  screenshots:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: node-    
+    - name: install
+      run: npm ci
+    - name: run
+      run: |
+        npm run watch & 
+        sleep 60s
+      env: 
+        APIS_IMG_SALT: ${{ secrets.APIS_IMG_SALT }}  
+        CAPI_KEY: ${{ secrets.CAPI_KEY }}
+    - name: Take screenshot of apps-rendering
+      uses: "lannonbr/puppeteer-screenshot-action@1.3.0"
+      with:
+        url: http://localhost:8080/
```

Comment:
Obviously we could capture a more interesting collection of screenshots here!

Reply:
Do you mean a different page, or a list of pages?

Reply:
I guess taking screenshots of several different articles would help us to catch more problems?